.github/workflows/modin.yml

```diff
@@ -35,8 +36,8 @@ jobs:
         with:
           path: |
             ~/.m2
-          key: ${{ env.RUN_STAMP }}-maven-${{ hashFiles('**/pom.xml') }}
-          restore-keys: ${{ env.RUN_STAMP }}-maven
+          key: maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: maven-
```

Comment:
@leshikus, I'm curious, why does the restore key not include the hash of the pom.xml?